crates/telio-sockets/src/protector/windows.rs

```diff
@@ -468,3 +468,28 @@ pub fn get_default_interface(tunnel_interface: u64) -> Result<Interface> {
 
     Ok(default_interface)
 }
+
+#[cfg(test)]
+mod tests {
+    use std::net::{IpAddr, Ipv4Addr, Ipv6Addr, SocketAddr, TcpListener, UdpSocket};
+
+    use crate::native::AsNativeSocket;
+    use rstest::rstest;
+
+    use super::*;
+
+    #[rstest]
+    #[case(IpAddr::V4(Ipv4Addr::LOCALHOST))]
+    #[case(IpAddr::V6(Ipv6Addr::LOCALHOST))]
+    #[tokio::test]
+    async fn test_make_external(#[case] ip_addr: IpAddr) {
```

Comment:
are we even running unit tests on windows?

Reply:
Yup, we do run them: https://github.com/NordSecurity/libtelio/actions/runs/6392487249/job/17349840328?pr=145#step:3:1201